src/Reports/LockedUsersReport.php

```diff
@@ -55,7 +57,17 @@ public function columns()
             'Email' => 'Email',
             'Locked' => 'Locked',
             'getGroupNames' => 'Groups',
-            'getLastAccessed' => 'Last Logged In',
+            'Date' => [
+                'title' => 'Last logged in',
+                'formatting' => function ($value, $item) {
+                    $customDate = $item->getLastLogin();
+                    $item->shouldBeLockedOut();
```

Comment:
That doesn't do anything does it?

Reply:
Yeah, looks like this can either be removed or maybe needs to be moved above the `getLastLogin` call

Reply:
Sorry that was a test